src/routes/landing.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import { Button, StyleSheet, Text, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+
+class Landing extends React.Component {
+  render() {
+    return (
+      <View style={styles.landing}>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+            <Button
+              title='Go To Search!!'
+              onPress={() => this.props.navigation.navigate('Search')}
+            />
+          </View>
+        </View>
+        <View style={styles.vContainer}>
+          <View style={styles.hContainer}>
+            <Text style={styles.text}>Almost half of the food in the U.S. goes to waste - approximately 3,000 pounds per second.</Text>
```

Comment:
Ah, forgot to mention that I have json data for about 100 recycling facts. We should cycle this info, but this is good for now.